ontologies/MIRO_sample_image_of_artwork.json

```diff
@@ -0,0 +1,108 @@
+; Comment: next step I assume would be to flesh out necessary context for this, including something 
+; to map these more human-readable terms to the ones in the .ttl ontologies
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "watu9tbc",
+  "type": "Work",
+  "identifiers": [
+    {
+      "label": "V0006724",
+      "authority": "MIRO"
+    },
+    {
+      "label": "ICV No 6939",
+      "authority": "Videodisk"
+    },
+    {
+      "label": ".b11579328",
+      "authority": "Sierra"
+    },
+  ],
+  "licence": "CC-BY-4.0",
+  "label": "A lecture on pneumatics at the Royal Institution, London. Co",
+  "description": "A lecture on pneumatics at the Royal Institution, London. Coloured etching by J. Gillray, 1802.",
+  "lettering": "Scientific researches! -New discoveries in pneumaticks! -or- an experimental lecture on the powers of air.- Js. Gillray inv. & fect.",
+  "creators": [
+      {
+      "label": "James Gillray",
+      "type": "Agent"
+      }
+      ],
+  "publishers": [
+      {
+      "label": "H. Humphrey,",
+      "type": "Agent"
+      }
+      ],
+  "placeofpublication": [
+      {
+      "label": "[London] (St. James's Street)",
+      "type": "Place"
+      }
+      ],
+  "createdDate": 
+    {
+      "label": "1802",
+      "type": "Period"
+    },
+  "publicationdate": 
+  {
+      "label": "May 23 1802",
+      "type": "Period"
+  },
+; Comment: MIRO data can include both a createdDate field generated by a MIRO cataloguer and a publicationDate field 
+; pulled across from Sierra (where appropriate): suggest where the latter exists it should take priority?
+  "format": "etching, with watercolour",
+  "physicalDescription": "Paper",
+  "genre": "Caricatures",
+  "subjects": 
+    [
+; Comment: I have not yet set up the Subjects ontology but am assuming that each subject will have a note of its label
+;  and its source authority.  In MIRO there are keywords that do not come from any external thesaurus - do we want to display these?
+      {
+        "label": "Caricatures",
+        "authority": "MeSH" 
+      },
+; Comment: given that the MIRO keywords are undisciplined and perhaps superseded by having better access to Sierra records
+; describing the source, do we even want to render them?   
+      {
```

Comment:
I'd think it would be useful to display and have MIRO keywords for searching if they aren't duplicating data from Sierra (especially for MIRO records that don't have a Sierra equivalent). I'm not clear from the comment whether MIRO keywords will always be bettered by Sierra data or not. If we don't know, we need data to decide, I guess.